byzcoin/collect_tx.go

```diff
@@ -112,6 +112,12 @@ func (p *CollectTxProtocol) Start() error {
 
 // Dispatch runs the protocol.
 func (p *CollectTxProtocol) Dispatch() error {
+	defer func() {
+		if r := recover(); r != nil {
+			log.Print(r)
+			log.Print(log.Stack())
```

Comment:
use `log.Error`
also: under what condition does `Dispatch` panic? is it not possible to fix the panic?

Reply:
Oups - debugging stuff. There was an error coming from `dedis/protobuf`, where the defer/recover in there tried to cast a value to an `error`, but it failed to do so. Unfortunately I cannot reproduce it anymore...